components/linkedin/actions/create-text-post-user/create-text-post-user.mjs

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+
+export default {
+  key: "linkedin_create-text-post-user",
+  name: "Create a Text Post (User)",
+  description: "Create shares on LinkedIn using text, URLs, and images. [See the docs](https://docs.microsoft.com/en-us/linkedin/consumer/integrations/self-serve/share-on-linkedin#creating-a-share-on-linkedin) for more information",
+  version: "0.0.11",
+  type: "action",
+  props: {
+    name: {
+      type: "string",
+      label: "Text",
+      description: "Text to be posted on LinkedIn timeline",
+    },
+    linkedin: {
+      label: "LinkedIn",
+      description: "LinkedIn Integration",
+      type: "app",
+      app: "linkedin",
+    },
+  },
+  async run({ $ }) {
+    const data = {
+      "author": `urn:li:person:${this.linkedin.$auth.oauth_uid}`,
```

Comment:
I would move this into the app file, as well, as a method that generates LinkedIn share payloads. We'll want to reuse this for other types of shares (e.g. images), where most of the payload will be the same.

Reply:
there is now in the linkedin.app.mjs two methods:

1. getPayload()
2. getRequestConfig()

Maybe the name should be changed to "generate" instead of "get"?